index.html

```diff
@@ -2,83 +2,103 @@
 <html>
     <head>
         <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>Innocent's Portfolio</title>
         <link rel="stylesheet" href="style.css">
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins:wght@400;700&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     </head>
-    <body class="container">
-        <section class="first-section">
-            <header>
-            <div class="nav">
-                <a href="#" class="nav-title">Innocent N.</a>
-                <nav>
-                    <img src="./images/menu.png" alt="menu-icon" class ="nav-menu" />
-                </nav>      
-            </div>
+    <body>
```

Comment:
- Please get the feedback from the original design in the [video](https://youtu.be/avnyFbbjxXI) 😉  I think you find this very helpful